### ISSUE TITLE
@ember/debug does not log messages to console

### ISSUE THREAD

**Issue by Redsandro** [emberjs/ember.js#18453]
`debug` from [`@ember/debug`](https://api.emberjs.com/ember/3.13/functions/@ember%2Fdebug/debug) does not log anything to the console when your build is run over `dnsmasq` i.e. has a domain name rather than an ip-address:port.

Is this a known issue, something worth looking into, or my own fault?
Did I mis some `environment.js` boolean?

```
ember-cli: 3.5.1
node: 8.16.0
```

```
node_modules/ember-cli/bin/ember s -dev
```

```
{
    "APP": {
        "LOG_ACTIVE_GENERATION": true, 
        "LOG_TRANSITIONS": true, 
        "LOG_TRANSITIONS_INTERNAL": false, 
        "LOG_VIEW_LOOKUPS": true, 
    }, 
    "EmberENV": {
        "EXTEND_PROTOTYPES": {
            "Date": false
        }, 
        "FEATURES": {}
    }, 
    "environment": "development", 
    "exportApplicationGlobal": true, 
    "locationType": "auto", 
    "rootURL": "/"
}
```

I'm using `console.log()` during development and manually remove them before production deploy. This is exactly what `debug()` is for (in theory), right?

**Comment by rwjblue** [comment]
I don't fully understand this issue. Using `import { debug } from '@ember/debug'` will do nothing in production builds, and in non-production builds it calls `console.debug` (if present) and falls back to `console.log`.

**Comment by Redsandro** [comment]
I did not know how this worked internally, I just used it for convenience.

The simplified issue is that `debug` commands do not work in my app anymore, although the app runs in `development` mode as you can see in the above `ENV`.

I assumed the change came from changing the development pipeline to use dnsmasq with ssl certificates (because of PWA requirements), but as you explained, that should not be relevant.

Do you have any ideas how I can trace why this is happening?

**Comment by rwjblue** [comment]
Well, does `console.debug` directly do anything? I'd test that...

**Comment by rwjblue** [comment]
FWIW, I don't really think this is a bug in Ember itself. Happy to keep chatting here, but I'm going to close this issue.

**Comment by Redsandro** [comment]
@rwjblue It didn't even occur to me to test this after you explained that internally it's simply a `console.debug()`. To my surprise, it didn't work either.

Only after looking in every menu, it turns out that this little guy was staring me directly in the face:

![image](https://user-images.githubusercontent.com/1702193/66271904-afc76f00-e863-11e9-9488-0e3c34c2a3b6.png)

The "default levels" needs to be "all levels". I guess I had set that some time ago, but it reverted to default (obviously) when I reinstalled my system.

I apologize profoundly for wasting your time.
